doc/conf.py

```diff
@@ -26,10 +28,20 @@
 copyright = u'2018, Umberto Villa, Noemi Petra, Omar Ghattas'
 author = u'Umberto Villa, Noemi Petra, Omar Ghattas'
 
+here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
This way we only need to maintain version in `__init__.py`, and both `setup.py` and `doc/conf.py` parse it from there as done here. Another common approach that I know of is to have `version.py` file somewhere, and import it in all three (init, conf, setup). It is not recommended to simply include `__init__.py` for the same purposes.

Reply:
Looks greats. But I see a 5 hard-coded.

Does it work both for release versions (like `2.1.1`) and development versions (like `2.2.0.dev`)?

Reply:
Whoops

Reply:
@eldarkh ,

Sorry, I just saw your suggestion of creating a `hippylib/version.py` file and directly import it in all three files. I think this is the most transparent, robust, and flexible solution.

Reply:
I see, what about the `Exec/execfile` solution described here?

https://milkr.io/kfei/5-common-patterns-to-version-your-Python-package

Reply:
I will need to look more into that since there is python 2 vs 3 differences with `execfile` if I am not mistaken, but looks promising also

Reply:
Looks like it works